zkevm-circuits/src/evm_circuit/execution/error_oog_call.rs

```diff
@@ -281,8 +305,16 @@ mod test {
                 accs[0]
                     .address(address!("0x000000000000000000000000000000000000cafe"))
                     .balance(Word::from(10u64.pow(19)));
-                accs[1].account(caller);
-                accs[2].account(callee);
+                accs[1]
+                    .address(caller.address)
+                    .code(caller.code.clone())
+                    .nonce(caller.nonce.as_u64())
+                    .balance(caller.balance);
+                accs[2]
+                    .address(callee.address)
+                    .code(callee.code.clone())
+                    .nonce(callee.nonce.as_u64())
+                    .balance(callee.balance);
```

Comment:
These were refactored away in #1409 to reduce the verbosity. Can we keep the simplified one, too?